ingress/controller.go

```diff
@@ -97,7 +97,8 @@ func (c *controller) updateLoadBalancer() error {
 	}
 
 	log.Infof("Updating load balancer with %d entry(s)", len(entries))
-	return c.lb.Update(entries)
+	_, err = c.lb.Update(LoadBalancerUpdate{entries})
```

Comment:
Isn't it a bit weird we ignore the return value? I guess it's used for tests only? Maybe we should log off of it in the controller at least.